src/core/api/index.js

```diff
@@ -116,7 +141,10 @@ export function callApi({
     }
   }
 
-  let apiURL = `${API_BASE}/${endpoint}/${queryString}`;
+  adjustedEndpoint = adjustedEndpoint.endsWith('/') ?
```

Comment:
Does this mess up any requests? I *think* the API always wants a trailing slash but I feel like we should double-check for any existing requests that don't have them and may be relying on the trailing slash being omitted.

I feel like that might be happening someplace? I know the API sometimes returns different errors for bad requests with/without the trailing slash so maybe that's why I'm thinking it matters, and it might not.

Reply:
The old code [always appended a slash](https://github.com/mozilla/addons-frontend/blob/a60c82a55cb3e53a1a57005e62992e6230489572/src/core/api/index.js#L119) to `endpoint` so that's not a new change.

This new code is just a bit more forgiving. If you make a helper that does `callApi({ endpoint: 'something/' })` it just works whereas before you'd end up with double slashes and your API request would always fail.